index.html

```diff
@@ -271,5 +271,59 @@ <h2 class="categorie-title">
     </ul>
 </section>
 </main>
+<footer class="footer-container" id="footer-container-id">
+  <div class="footer-form">
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form action="https://formspree.io/f/xzbovlzk" method="post">
+      <ul>
+        <li>
+          <input type="text" name="Name" placeholder="Full name">
+        </li>
+        <li>
+          <input type="email" name="Email" placeholder="Email address">
+        </li>
+        <li>
+          <textarea name="Message" id="msg" cols="30" rows="10">Enter text here</textarea>
+        </li>
```

Comment:
- [ ] According to the project requirement: 

- Name field must accept up to 30 chars, which means, the Name input should have the `maxlength="30"` attribute.
- Message field accepts up to 500 chars which means, the Message input should have the `maxlength="500"` attribute.